src/monitormanager.cpp

```diff
@@ -217,58 +217,42 @@ void MonitorManager::removeMonitor(HSMonitor* monitor)
 int MonitorManager::addMonitor(Input input, Output output)
 {
     // usage: add_monitor RECTANGLE [TAG [NAME]]
-    if (input.size() < 2) {
+    std::string cmd, rectString, tagName, monitorName;
+    input >> cmd >> rectString;
+    if (!input) {
         return HERBST_NEED_MORE_ARGS;
     }
-
-    input.shift();
-    auto rect = Rectangle::fromStr(input.front());
-
-    input.shift();
-    HSTag* tag;
-    if (input.empty()) {
-        tag = find_unused_tag();
-        if (!tag) {
-            output << input.command() << ": There are not enough free tags\n";
-            return HERBST_TAG_IN_USE;
-        }
-    } else {
-        tag = find_tag(input.front().c_str());
+    HSTag* tag = NULL;
```

Comment:
Use `nullptr` for a stupidly tiny bit of type safety?

Reply:
I've changed this to `nullptr`.